### PR TITLE
Redirect What's new in C# to C# 12, not 11

### PR DESCRIPTION
**Pull request by BillWagner**
We've got preview content now

### CODE REVIEW

**Comment by BillWagner** [comment]
ping @adegeo for review

File: .openpublishing.redirection.csharp.json [hunk 1]
```diff
@@ -3130,7 +3130,7 @@
         },
         {
             "source_path_from_root": "/docs/csharp/whats-new.md",
-            "redirect_url": "/dotnet/csharp/whats-new/csharp-11",
+            "redirect_url": "/dotnet/csharp/whats-new/csharp-12
```
Comment (tdykstra) [inline .openpublishing.redirection.csharp.json:3133]:
```suggestion
            "redirect_url": "/dotnet/csharp/whats-new/csharp-12"
```

**Review by tdykstra — APPROVED** [review]
Missing a closing quote

File: .openpublishing.redirection.csharp.json [hunk 2]
```diff
@@ -3130,7 +3130,7 @@
         },
         {
             "source_path_from_root": "/docs/csharp/whats-new.md",
-            "redirect_url": "/dotnet/csharp/whats-new/csharp-11",
+            "redirect_url": "/dotnet/csharp/whats-new/csharp-12"
```
Comment (BillWagner) [inline .openpublishing.redirection.csharp.json:3133]: need the comma:

```suggestion
            "redirect_url": "/dotnet/csharp/whats-new/csharp-12",
```

**Pull request merged**